charts/rancher-turtles/Chart.yaml

```diff
@@ -23,7 +23,7 @@ annotations:
   catalog.cattle.io/namespace: rancher-turtles-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux
-  catalog.cattle.io/rancher-version: '>= 2.8.1-0'
+  catalog.cattle.io/rancher-version: '>= 2.9.0-1'
```

Comment:
`2.9.0` is rancher version, but what does `-1` define in this format?

Reply:
This comes from the ADR that defines these annotations and specifies that there needs to be a lower and upper bound:

> There are two annotations: `catalog.cattle.io/rancher-version` and `catalog.cattle.io/kube-version` which must include lower and upper bounds.